### DISCUSSION TITLE
Setting up home with this plugin

### DISCUSSION THREAD

**Discussion by Pseudomizer** [RobertD502/home-assistant-flair#50] [General]
Hello Robert and all,

I am trying to figure out how to achieve my goal for my specific home. My challenge is the following and maybe you can advise on the settings of your plugin as your plugin seems to have the flexibility.

Two floors. 
Upper floor is always too hot in the summer (3 rooms) and downstairs is nice and cool.
Downstairs is always too cold in the winter (3 rooms) and upstairs gets too warm trying to warm up downstairs.

I have 6 rooms. I have pucks in every room and I also have Ecobee sensors. I have 3 vents downstairs and 3 vents upstairs. 

In Flair you have the option to set the thermostat to control the vents aka Ecobee or you can let Flair control them. That is the auto mode. As it looks like when you set temperatures with the puck, it applies a range for each room, which cannot be changed meaning if you set it to 73 then the vent will stay open until the room reaches 74 for heating up when the room has 69 degrees to begin with.

I haven't tried the air condition yet, as I just got the vents.

With your plugin, I would like to create two groups downstairs and upstairs. I would like to set the temperature for upstairs to 72 and downstairs to 70 and keep that temperature throughout because without those vents the temperature difference between the floors is about 10 to 12 degrees meaning if it is upstairs 74, we have downstairs 64 to 62. 

That's the basic function. Now to make this a real smart home, with the occupancy sensors in the room you can let downstairs cool down over night if nobody is present meaning it should close the vents downstairs despite being too cold and only heat upstairs. 

In the summer the reverse is true. Upstairs is usually way too warm so we stay downstairs. If nobody is up there, we don't want the air condition to kick in and cool upstairs unless somebody goes up there. 

If you would have to use your plugin, how would you accomplish this goal? Any hints are welcome and I just got those vents, so I am trying to figure this all out. 

Thanks in advance.

**Comment by seanmahrt** [comment]
The hysteresis in auto mode seems fixed at +/- 1F from what I can tell.  Are you looking for two set points basically?  Or are you really only after 1 temperature?  you could put all the downstairs vents in one "room" and tie it to one temp sensor to make them all move together.

I think the flair auto mode has an occupancy option too.  I've never played with it as my heat/cool probably won't move fast enough for the occupancy sensor to help per room.

Ecobee does have the occupancy trigger so you could control when to run the heat/ac based on occupancy.

I'm not sure you would even need to do black magic within HA to do it, but I am likely not understanding it completely.
